src/css/work-together.css

```diff
@@ -0,0 +1,327 @@
+.work-together {
+    max-width: 360px;
+    height: 661px;
+    padding-left: 20px;
+    padding-right: 20px;
+}
+
+.title-headline {
+    font-weight: 400;
+    font-size: 52px;
+    line-height: 123%;
+    letter-spacing: 0.03em;
+    text-transform: uppercase;
+    color: black;
+}
+
+.title-together {
+    min-width: 253px;
+    margin-bottom: 57px;
+}
+
+.green-text {
+    color: #00B068;
+}
+
+.title-text {
+    font-family: var(--font-family);
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 156%;
+    letter-spacing: 0.03em;
+    text-transform: uppercase;
+    color: black;
+    margin-bottom: 32px;
+}
+
+.title-contacts {
+    font-family: var(--font-family);
+    font-weight: 300;
+    font-size: 16px;
+    line-height: 150%;
+    letter-spacing: 0.03em;
+    color: black;
+
+}
+
+.contacts-together {
+    width: 258px;
+    height: 248px;
+
+
+}
+
+.form-message {
+    width: 100%;
+    height: 216px;
```

Comment:
Please, get rid of `height` property. Try to avoid `height` property. Because height determined by content. It is a bad practice when you set fix `height`